hathor/merged_mining/coordinator.py

```diff
@@ -502,6 +512,7 @@ def handle_error(self, error: Dict, data: Any, msgid: Optional[str]) -> None:
     def handle_authorize(self, params: List, msgid: Optional[str]) -> None:
         """ Handles authorize request by always authorizing even if the request is invalid.
         """
+        # XXX
```

Comment:
Missing comment.

Reply:
I was using this as a placeholder. Removed.